priv/templates/solaris/Makefile

```diff
@@ -50,7 +50,9 @@ prototype:
 	echo "i r.preserve" >> prototype
 	echo '' >> prototype
 	pkgproto rel/{{package_install_name}}={{package_install_name}} >> prototype
-	sed -e "s/ $(LOGNAME) .*$$/ {{package_install_user}} {{package_install_group}}/" \
+	sed -e "s/ $(LOGNAME) .*$$/ root bin/" \
```

Comment:
Yea - I certainly didn't reduce the confusion ratio here, but the problem is the pkgproto tool just spits out the information to the prototype file, so we've got to go muck with it later. Something along the lines of "By default, all files should be owned by root/bin. However, `log` and `data` directories much be writable by the package owner/group" above the sed script might help?